app/controllers/dashboard_controller.rb

```diff
@@ -85,6 +120,39 @@ def select
     end
   end
 
+  def getHangoutsLink
+    query = Project.where(pivotal_id: params[:project_id].to_s)
+    if (!query.any?) 
+      proj = Project.new
```

Comment:
This seems like a class method of `Project`.

Reply:
You're right. Refactored.